ext/opcache/jit/zend_jit_internal.h

```diff
@@ -446,6 +446,8 @@ typedef struct _zend_jit_trace_exit_info {
 	int32_t              poly_this_ref;
 	int8_t               poly_func_reg;
 	int8_t               poly_this_reg;
+	int32_t              call_ref;
+	int8_t               call_reg;
```

Comment:
Deoptimization relied on `EX(call)` being in IP when `ZEND_JIT_EXIT_RESTORE_CALL` was set. I now use the SNAPSHOT mechanism so that deopt knows in which register the call actually is.